cluster/gce/gci/configure.sh

```diff
@@ -723,7 +658,7 @@ function install-kube-binary-config {
 # and sets the HOST_PLATFORM and HOST_ARCH environment variables accordingly.
 # Callers can specify HOST_PLATFORM_OVERRIDE and HOST_ARCH_OVERRIDE to skip the detection.
 # This function is adapted from the detect_client_info function in cluster/get-kube-binaries.sh
-# and kube::util::host_os, kube::util::host_arch functions in hack/lib/util.sh
+# and kube::util::host_os, kube::util::host_arch functions in cluster/util.sh
```

Comment:
If I missed review of any place - we need to keep `cluster/util.sh` instead of `hack/lib/util.sh`